docs/elements/quickbooksonline/tips.md

```diff
@@ -0,0 +1,31 @@
+---
+heading: QuickBooks Online
+seo: Tips | QuickBooks Online | Cloud Elements API Docs
+title: Tips
+description: QuickBooks Online Tips
+layout: sidebarelementdoc
+breadcrumbs: /docs/elements.html
+elementId: 39
+parent: Back to Element Guides
+order: 75
+---
+
+# Tips
+
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
+
+{% include callout.html content="<strong>On this page</strong><br/><a href=#general>General</a><br/><a href=#authentication>Authentication</a>" type="info" %}
+
+
+## General
+
+* Quickbooks Online does not allow you to update taxes via third-party service, including API calls. If you need this functionality, delete the transaction and re-create it using the correct taxes.
+* Quickbooks Online does not currently support the "OR" operator in the where/search clause.
+* When you GET/ Records, Quickbooks Online returns a sync token in the format of (22|33). The first number represets the ID, the second represents the sync token, which is not a static value. To update a record, you must provide the most recent sync token.  If you try to update a record with an old sync token, you will get an error message. It is best practice to retrieve the record right before you update to ensure that you have the latest sync token.
```

Comment:
add tics around GET /Records... also, confirm that shouldn't be lowercase records.

typo in represents (2nd sentence).

Look like extra space between 4rd and 4th sentences